envi/archs/arm/emu.py

```diff
@@ -11,6 +12,8 @@
 from envi.archs.arm.regs import *
 from envi.archs.arm import ArmModule
 
+log = logging.getLogger(__name__)
```

Comment:
please make this variable "logger" ( to align with the new convention )

with that final delta, i'll mainline this PR :)